flask_project/campaign_manager/models/models.py

```diff
@@ -0,0 +1,541 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import os
+
+from sqlalchemy import (
+    create_engine,
+    Column,
+    Integer,
+    String,
+    Table,
+    Text,
+    ForeignKey,
+    Date,
+    DateTime,
+    BOOLEAN
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from geoalchemy2 import Geometry
+
+
+db_location = os.environ['DATABASE_URL']
+engine = create_engine(db_location, echo=True)
+
+
+Base = declarative_base()
+
+adminAssociations = Table(
+    'adminAssociations',
+    Base.metadata,
+    Column(
+        'user_id',
+        Integer,
+        ForeignKey('user.id_user'),
+        primary_key=True
+        ),
+    Column(
+        'campaign_id',
+        Integer,
+        ForeignKey('campaign.id_campaign'),
+        primary_key=True
+        )
+    )
+
+typeCampaignAssociations = Table(
+    'typeCampaignAssociations',
+    Base.metadata,
+    Column(
+        'campaign_id',
+        Integer,
+        ForeignKey('campaign.id_campaign'),
+        primary_key=True
+        ),
+    Column(
+        'type_id',
+        Integer,
+        ForeignKey('featureType.id_type'),
+        primary_key=True
+        )
+    )
+
+featureTypeAssociations = Table(
+    'featureTypeAssociations',
+    Base.metadata,
+    Column(
+        'feature_id',
+        Integer,
+        ForeignKey('attribute.id_feature'),
+        primary_key=True
+        ),
+    Column(
+        'type_id',
+        Integer,
+        ForeignKey('featureType.id_type'),
+        primary_key=True
+        )
+    )
+
+teamUserAssociations = Table(
+    'teamUserAssociations',
+    Base.metadata,
+    Column(
+        'user_id',
+        Integer,
+        ForeignKey('user.id_user'),
+        primary_key=True
+        ),
+    Column(
+        'team_id',
+        Integer,
+        ForeignKey('team.id_team'),
+        primary_key=True
+        )
+    )
+
+functionCampaignAssociations = Table(
+    'functionCampaignAssociations',
+    Base.metadata,
+    Column(
+        'campaign_id',
+        Integer,
+        ForeignKey('campaign.id_campaign')
+        ),
+    Column(
+        'function_id',
+        Integer,
+        ForeignKey('function.id_function')
+        )
+    )
+
+
+class User(Base):
+
+    __tablename__ = 'user'
```

Comment:
It doesnt really matter but I would use plural name for SQL tables.
user -> users
campaign -> campaigns